source/core_sntp_client.c

```diff
@@ -298,7 +295,7 @@ static SntpStatus_t addClientAuthentication( SntpContext_t * pContext )
     if( status != SntpSuccess )
     {
         LogError( ( "Unable to send time request: Client authentication function failed: "
-                    "RetStatus=%u", status ) );
+                    "RetStatus=%s", StatusToStr( status ) ) );
```

Comment:
I don't think this is a function
```suggestion
                    "RetStatus=%s", Sntp_StatusToStr( status ) ) );
```

Reply:
Have enabled logging in unit test build to catch this form of error.